pom.xml

```diff
@@ -498,6 +498,7 @@
           <consoleOutput>true</consoleOutput>
           <excludes>
```

Comment:
I am pretty sure we can change `excludes` for `excludesFile` (https://creadur.apache.org/rat/apache-rat-plugin/rat-mojo.html#excludesFile) but I am unsure why some of the regex on the individual excludes don't seem to work with excludesFile when I ran:
`java  -jar apache-rat-0.16.1/apache-rat-0.16.1.jar -a -d apache-parquet-1.15.0.tar.gz -E $PARQUET_SRC_FOLDER/.rat-excludes.txt`
I'll investigate how to consolidate those two lists